src/templates/index.js

```diff
@@ -1,3 +1,3 @@
 import * as bootstrap from "./bootstrap";
 
-export default { bootstrap };
+export default { bootstrap, bootstrap3: bootstrap };
```

Comment:
Can you explain how this would work.

Would you not need to provide a key for the original one.

Ie,
bootstrap: bootstrap, bootstrap3: bootstrap

Reply:
indeed
{ bootstrap, bootstrap3: bootstrap }
is a ES6 shorthand for
{ bootstrap: bootstrap, bootstrap3: bootstrap }